src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -177,12 +177,18 @@ internal static bool DoSslHandshake(SafeSslHandle context, byte[] recvBuf, int r
         {
             sendBuf = null;
             sendCount = 0;
+            int retVal;
             if ((recvBuf != null) && (recvCount > 0))
             {
-                BioWrite(context.InputBio, recvBuf, recvOffset, recvCount);
+                retVal = BioWrite(context.InputBio, recvBuf, recvOffset, recvCount);
+                if(retVal <= 0)
```

Comment:
nit: space after if

Reply:
While I agree that this is matching the previous behavior (ignoring the errors) it seems like they should be handled (if handleable... but since it's memory BIOs I don't think there's anything for "try again") or an exception thrown.

*Maybe* the reads have cases where they make sense to ignore an error, but the write definitely assumes that the whole buffer gets written out.

Heck, an exception should probably be thrown if `retVal != recvCount`, which could be different than the 0/-1 case of checking for an error.